src/main/java/cdw/springtraining/gatekeeper/entites/User.java

```diff
@@ -0,0 +1,40 @@
+package cdw.springtraining.gatekeeper.entites;
+
+import jakarta.persistence.*;
```

Comment:
import only needed package and avoid importing all packages

Reply:
Removed WIldcard Entries